components/dashboard/src/Menu.tsx

```diff
@@ -275,22 +271,22 @@ export default function Menu() {
     const onFeedbackFormClose = () => {
         setFeedbackFormVisible(false);
     };
-    const isTeamLevelActive = !projectSlug && !isWorkspacesUI && !isPersonalSettingsUI && !isAdminUI && teamOrUserSlug;
+    const isTeamLevelActive = !projectSlug && !isWorkspacesUI && !isPersonalSettingsUI && !isAdminUI;
     const renderTeamMenu = () => {
         if (!hasIndividualProjects && (!teams || teams.length === 0)) {
             return (
                 <div className="p-1 text-base text-gray-500 dark:text-gray-400 border-gray-800">
                     <PillMenuItem
                         additionalClasses="border-2 border-gray-200 dark:border-gray-700 border-dashed"
-                        name="New Team →"
-                        link="/teams/new"
+                        name="New Organization →"
+                        link="/orgs/new"
                     />
                 </div>
             );
         }
         const userFullName = user?.fullName || user?.name || "...";
-        const entries: (ContextMenuEntry & { slug: string })[] = [
-            ...(hasIndividualProjects
+        const entries: ContextMenuEntry[] = [
+            ...(!user?.additionalData?.isMigratedToTeamOnlyAttribution
```

Comment:
with this change I'm adding the previously deprecated  "personal account" entry back in the drop-down menu unless you have been migrated (in preview env everyone is migrated, so you don't see this) If you want to test the behavior you need to change the config cat value and delete/recreate your user.